### PR TITLE
Multiple datasets and configuration files

### LINKED ISSUE #12: Select mutations from mutation table instead of list
For plots that segregate by mutational status, populate the list of possible mutations from the mutation db table

### PR DESCRIPTION
**Pull request by paytonredemer**
I added tcga_aml and beat_aml datasets. Also, the basic idea of configuration files is implemented to display only the available plots for each dataset.

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -1,253 +1,132 @@
...
+
+        query <- clinicalQuery(factors = "*")
+        clinical <- dbGetQuery(database,query)
+        clinical <- merge(tcga, clinical, by="UPN")
+
+        g <- ggplot(clinical,aes(fill=Gene, y=Expression, x=UPN, text = paste0("UPN ID: ",UPN,"<br />Dataset: ", Short_hand_code))) + geom_bar(position="dodge", stat="identity") + theme_bw() +
+          theme(text=element_text(size=12, family="avenir", face="bold"), axis.text=element_text(size=10, family="avenir", face="bold"),
+                axis.title=element_text(size=12, family="avenir", face="bold"),
+                axis.text.x = element_text(angle = 90, hjust = 1)) +
+          ggtitle("Multiple protein view") +
```
Comment (chrisamiller) [inline server.R]: This title should drop "Protein" to be generic

File: global.R [hunk 2]
```diff
@@ -10,19 +10,24 @@ source("functions.R")
...
+proteomics <- dbConnect(RSQLite::SQLite(), "Proteomics.db")
+database <- dbConnect(RSQLite::SQLite(), "master_shiny_app.db")
 print("Connected to database")
 
 # database vars
 # choices are only loaded once per server rather than every client
-TMT_choices <- unique(dbGetQuery(database, "SELECT Gene FROM Genes WHERE Type='TMT';"))
-LFQ_choices <- unique(dbGetQuery(database, "SELECT Gene FROM Genes WHERE Type='LFQ';"))
-phosphosite_choices <- dbGetQuery(database, "SELECT Gene FROM Genes WHERE Type='Phosphosite';")
+TMT_choices <- unique(dbGetQuery(proteomics, "SELECT Gene FROM Genes WHERE Type='TMT';"))
```
Comment (chrisamiller) [inline global.R]: I think we decided that these were all unused now, right?

**Review by chrisamiller — COMMENTED** [review]
Made two quick comments, and I know you said there's still a display bug. Once that's ironed out, I think we can launch this as an alpha version!

**Comment by chrisamiller** [comment]
I believe this closes https://github.com/leylabdotorg/amlviz/issues/12

**Comment by paytonredemer** [comment]
I decided to start using the shinyjs library to toggle the visibility of UI elements. I tried doing it with native shiny functions, but they were too limiting. I also tried plain javascript, but at that point, I was reimplementing shinyjs.

**Review by chrisamiller — COMMENTED** [review]
Nice - I can see how shinyjs simplifies some things.  My only comment is the "Multiple protein view" thing I pointed out last time

**Comment by paytonredemer** [comment]
> Nice - I can see how shinyjs simplifies some things. My only comment is the "Multiple protein view" thing I pointed out last time

Changed for now, but I'll make it based on the db later.

**Pull request merged**